setup.py

```diff
@@ -43,28 +43,27 @@
         long_description = f.read()
 
     install_requirements = [
+        "vivarium_dependencies[pandas,scipy]",
         "vivarium_build_utils>=2.0.1,<3.0.0",
         "numpy",
```

Comment:
For these repos that have unpinned `numpy`, once we unpin numpy, we're going to want to change these to use the `vivarium_dependencies[numpy]`